### PR TITLE
Only Display Errors in Properties Panel

### PR DESCRIPTION
**Pull request by philippfromme**
brave_wZs3wNvX1j

Depends on https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/137

Related to https://github.com/camunda/camunda-modeler/issues/3834

### CODE REVIEW

File: test/spec/utils/properties-panel.spec.js [hunk 1]
```diff
@@ -1759,6 +1760,25 @@ describe('utils/properties-panel', function() {
         });
       });
 
+
+      it('should not return errors for category info', async function() {
```
Comment (nikku) [inline test/spec/utils/properties-panel.spec.js:1764]: Let's add another test for `warn` category and we're good.

File: lib/utils/properties-panel.js [hunk 2]
```diff
@@ -384,7 +384,11 @@ export function getErrorMessage(id, report) {
     return 'Type must be defined.';
   }
 
-  if (id === 'messageSubscriptionCorrelationKey') {
+  if (id === 'messageSubscriptionCorrelationKey'
+    && [
+      ERROR_TYPES.EXTENSION_ELEMENT_REQUIRED,
```
Comment (nikku) [inline lib/utils/properties-panel.js:389]: How is this related to the PR description?

Comment (philippfromme) [reply lib/utils/properties-panel.js:389]: The check against the ID of the properties panel entry is now too generic since there is more than one error type for the _Subscription correlation key_ entry. We have to do the same for [other entries](https://github.com/camunda/linting/blob/main/lib/utils/properties-panel.js#L447) as well.

File: lib/utils/properties-panel.js [hunk 3]
```diff
@@ -298,10 +302,6 @@ export function getErrorMessage(id, report) {
     return 'Cannot be an expression.';
   }
 
-  if (data.type === ERROR_TYPES.SECRET_EXPRESSION_FORMAT_DEPRECATED) {
-    return 'Secret expression format deprecated.';
```
Comment (nikku) [inline lib/utils/properties-panel.js:302]: Dropped and will be re-added?

**Review by nikku — CHANGES_REQUESTED** [review]
Missing (to my knowledge) - Test to verify warn does not propagate to properties panel.

File: lib/utils/properties-panel.js (hunk 3 above)
Comment (philippfromme) [reply lib/utils/properties-panel.js:302]: Since we don't want to show an error in the properties panel this code is useless.

**Comment by philippfromme** [comment]
> Missing (to my knowledge) - Test to verify warn does not propagate to properties panel.

See https://github.com/camunda/linting/pull/86/files#diff-9e46d7301b89e189b8ecc9b3149b778ce3707b220ab9e243e0ce07735d02fc67R1783.

File: test/spec/utils/error-messages.spec.js [hunk 4]
```diff
@@ -63,7 +63,7 @@ describe('utils/error-messages', function() {
           const errorMessage = getErrorMessage(report, 'Camunda Cloud', executionPlatformVersion);
 
           // then
-          expect(errorMessage).to.equal('A <Signal Start Event> in a <Sub Process> is not supported by Camunda 8.2');
+          expect(errorMessage).to.equal('A <Signal Start Event> in a <Sub Process> is only supported by Camunda 8.3 or newer');
```
Comment (nikku) [inline test/spec/utils/error-messages.spec.js:66]: :star:

**Review by nikku — APPROVED** [review]
